whois-nrtm4/src/main/java/net/ripe/db/nrtm4/dao/SourceRepository.java

```diff
@@ -0,0 +1,41 @@
+package net.ripe.db.nrtm4.dao;
+
+import net.ripe.db.nrtm4.domain.NrtmSource;
+import net.ripe.db.nrtm4.domain.NrtmSourceModel;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.sql.PreparedStatement;
+import java.sql.Statement;
+
+
+@Repository
+public class SourceRepository {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    SourceRepository(
+        @Qualifier("nrtmDataSource") final DataSource dataSource
+    ) {
+        this.jdbcTemplate = new JdbcTemplate(dataSource);
+    }
+
+    public NrtmSourceModel createSource(final NrtmSource source) {
+        final KeyHolder keyHolder = new GeneratedKeyHolder();
+        jdbcTemplate.update(connection -> {
+            final String sql = """
+                INSERT INTO source (name)
+                VALUES (?)
+                """;
+            final PreparedStatement pst = connection.prepareStatement(sql, Statement.RETURN_GENERATED_KEYS);
+            pst.setString(1, source.name());
```

Comment:
Do you really need to pass on Statement.RETURN_GENERATED_KEYS? It should be automatically set to 1?